# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/mochitests/file_testloadflags.js
# Commit: 532731e94bb2
# Full Hash: 532731e94bb23b01b5e7136bee829844a0f08bc2
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2020-06-01 21:42:28
# Regressor Bug: 1638358
# File Overlap Count: 1
# Description:
#   Bug 1638358 - Cookie Schemeful Same-Site - part 5 - schemeful comparison, r=mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D75629
# ==============================================================================

diff -r fad2ba760157 -r 532731e94bb2 netwerk/test/mochitests/file_testloadflags.js
--- a/netwerk/test/mochitests/file_testloadflags.js	Mon Jun 01 11:43:30 2020 +0000
+++ b/netwerk/test/mochitests/file_testloadflags.js	Mon Jun 01 11:43:28 2020 +0000
@@ -36,7 +36,12 @@
 
   var prefSet = new Promise(resolve => {
     SpecialPowers.pushPrefEnv(
-      { set: [["network.cookie.cookieBehavior", 1]] },
+      {
+        set: [
+          ["network.cookie.cookieBehavior", 1],
+          ["network.cookie.sameSite.schemeful", false],
+        ],
+      },
       resolve
     );
   });